packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -42,6 +41,19 @@ export const TokensList: React.FC<TokensListProps> = (props) => {
     setFilterString(filter);
   };
 
+  const handleThemeChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    // @ts-ignore todo: figure out ts error with DarkModeTokens
+    if (e.currentTarget.value === 'dark') setTokens(DarkModeTokens.tokens);
```

Comment:
need some help debugging this ts error on `setTokens(DarkModeTokens.tokens)`

Argument of type '{ "background-colors": [{ type: "color"; category: "background-color"; value: "rgb(18, 28, 45)"; comment: "Default background color for any container"; originalValue: "{!palette-gray-100}"; name: "color-background"; }, ... 49 more ..., { ...; }]; ... 12 more ...; "z-indices": [...]; }' is not assignable to parameter of type 'SetStateAction<{ "background-colors": [{ type: "color"; category: "background-color"; value: "rgb(244, 244, 246)"; comment: "Background color used for containers."; originalValue: "{!palette-gray-10}"; name: "color-background"; }, ... 70 more ..., { ...; }]; ... 12 more ...; "z-indices": [...]; }>'.
  Type '{ "background-colors": [{ type: "color"; category: "background-color"; value: "rgb(18, 28, 45)"; comment: "Default background color for any container"; originalValue: "{!palette-gray-100}"; name: "color-background"; }, ... 49 more ..., { ...; }]; ... 12 more ...; "z-indices": [...]; }' is not assignable to type '{ "background-colors": [{ type: "color"; category: "background-color"; value: "rgb(244, 244, 246)"; comment: "Background color used for containers."; originalValue: "{!palette-gray-10}"; name: "color-background"; }, ... 70 more ..., { ...; }]; ... 12 more ...; "z-indices": [...]; }'.
    Types of property '"background-colors"' are incompatible.
      Type '[{ type: "color"; category: "background-color"; value: "rgb(18, 28, 45)"; comment: "Default background color for any container"; originalValue: "{!palette-gray-100}"; name: "color-background"; }, ... 49 more ..., { ...; }]' is not assignable to type '[{ type: "color"; category: "background-color"; value: "rgb(244, 244, 246)"; comment: "Background color used for containers."; originalValue: "{!palette-gray-10}"; name: "color-background"; }, ... 70 more ..., { ...; }]'.
        Source has 51 element(s) but target requires 72.ts(2345)